[16.0][ADD] sale_stock_available_from_route_location (#2)
QuocDuong1306/stock-logistics-availability: sale_stock_route_location/readme/DESCRIPTION.rst

> Currently, the forecasted and available quantities on `Availability` widget of Sale Order are based on all locations of the warehouse

Reviewer: ```suggestion
Currently, the forecasted and available quantities on `Availability` widget of Sale Order are based on all locations of the warehouse.
```

> This module will re-calculate those quantities and base on locations defined on `route_id` of Sale Order Line

Reviewer: ```suggestion
This module will re-calculate those quantities based only on the locations defined for the defined route of the Sale Order Line
```
